.github/actions/rust-build-release/tests/test_manifest_path.py

```
@@ -0,0 +1,287 @@
+
+from __future__ import annotations
+
+import collections.abc as cabc
+import typing as typ
+from pathlib import Path
+from types import ModuleType
+
+import pytest
+from plumbum.commands.processes import ProcessExecutionError
+
+# ruff: noqa: D103
```

Ruff (0.14.3)

Unused `noqa` directive (non-enabled: `D103`)

Remove unused `noqa` directive

(RUF100)